ui/src/app/layout/applications/application-block.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, EventEmitter, Input, OnChanges, Output, SimpleChanges } from '@angular/core';
+import { UserApplicationInfoWithTopics } from './applications.component';
+import { combineLatest, Observable, of } from 'rxjs';
+import { catchError, map, shareReplay, startWith } from 'rxjs/operators';
+import { ApplicationCertificate, CertificateService } from '../../shared/services/certificates.service';
+import { EnvironmentsService, KafkaEnvironment } from '../../shared/services/environments.service';
+import { TranslateService } from '@ngx-translate/core';
+import * as moment from 'moment';
+import { ReplayContainer } from '../../shared/services/services-common';
+
+export interface OpenCertificateDialogEvent {
+    application: UserApplicationInfoWithTopics;
+
+    environment: KafkaEnvironment;
+}
+
+@Component({
+    selector: 'app-application-block',
+    templateUrl: './application-block.component.html',
+    styleUrls: ['./application-block.component.scss']
+})
+export class ApplicationBlockComponent implements OnChanges {
```

Comment:
I think it would be good idea to test this new Component :) Maybe we could ensure that the correct ApplicationPrefixes are present in the table.

Reply:
Discussed via Chat: Test will be added after merge due to time restrictions.